src/main/java/org/openstreetmap/atlas/checks/base/BaseCheck.java

```diff
@@ -259,6 +307,14 @@ protected CheckFlag createFlag(final Set<AtlasObject> objects, final String inst
 
     protected abstract Optional<CheckFlag> flag(AtlasObject object);
 
+    /**
+     * Method to implement for inheriting checks to return the default set of instruction formats
+     * that will be the last resort in {@link BaseCheck#getLocalizedInstruction(int, Object[])}
+     *
+     * @return The set of instructions to fall back to if configuration results in none.
+     */
+    protected abstract List<String> getFallbackInstructions();
```

Comment:
Let's implement this as a default method and return empty list. I don't think we need to force checks to implement a fallback instruction as this is the last resort to get a instruction.

Reply:
On principle I certainly agree, however currently most of the instructions are set in the check itself, so unless we plan on enforcing the instructions to be set through configuration I think we have to enforce the check to have a fallback in the code. If we do enforce it in configuration then we could just get rid of this function altogether. Either way I think this is a good discussion to have but maybe not for this PR.